### ISSUE TITLE
Add RPC to enumerate contract methods

### ISSUE THREAD

**Issue by vgrichina** [nearprotocol/nearcore#274]

**Comment by evgenykuzyakov** [comment]
@vgrichina We need to expose some view call method that would describe the contract first. Then we can just return this to the nearlib. It's possible we don't even need to implement any RPC changes

**Comment by vgrichina** [comment]
@evgenykuzyakov yeah, I think we only need compiler and bindgen changes. On `nearcore` side we might need some changes to enforce annotations though (e.g. if we want to mark methods than can spend money).

**Comment by ilblackdragon** [comment]
Should be part of https://github.com/nearprotocol/NEPs/pull/3 work

**Comment by bowenwang1996** [comment]
What should the return format be? @vgrichina @evgenykuzyakov

**Comment by bowenwang1996** [comment]
Need to have a design first. Moving to NEPs
